app/assets/javascripts/discourse/app/components/topic-map/topic-participant.gjs

```diff
@@ -0,0 +1,72 @@
+import Component from "@glimmer/component";
+import { htmlSafe } from "@ember/template";
+import UserAvatarFlair from "discourse/components/user-avatar-flair";
+import { userPath } from "discourse/lib/url";
+import { avatarImg } from "discourse-common/lib/avatar-utils";
+import gt from "truth-helpers/helpers/gt";
+
+let addTopicParticipantClassesCallbacks = [];
+
+export function addTopicParticipantClassesCallback(callback) {
+  addTopicParticipantClassesCallbacks =
+    addTopicParticipantClassesCallbacks || [];
```

Comment:
This variable is initialized as `[]`, and there is nothing which would set it to anything else. So I think we can drop this?

```suggestion
```